src/Scripts/UI/Controls/GraphicsEditor.gd

```diff
@@ -4,6 +4,7 @@ extends PanelContainer
 
 signal masking_ended()
 
+var Buble = preload("res://Scenes/CloudControl.tscn")
```

Comment:
it could be a good idea to change the name to bubble_scene so its less confusing